core/test/math/bigint_tests.cpp

```diff
@@ -176,4 +176,10 @@ TEST(BigInt, FromFloatString) {
     auto t = witness.toByteArray();
     auto tt = subject.toByteArray();
     EXPECT_TRUE(witness.compare(subject) == 0);
+}
+
+TEST(BigInt, AssignFromUnsignedScalar) {
+    uint64_t value = 0xFFFFFFFFFFFFFF01UL;
+    BigInt bigInt = BigInt::fromScalar(value);
+    EXPECT_EQ(value, bigInt.toUint64());
```

Comment:
There is a test somewhere for signed scalar ?

Reply:
Indeed there is no direct test, I add one